packages/react-docgen/src/utils/__tests__/findFunctionReturn-test.ts

```diff
@@ -152,6 +152,27 @@ describe('findFunctionReturn', () => {
       expect(findFunctionReturn(def, predicate)).toBeUndefined();
     });
 
+    test('handles direct prop return', () => {
+      const def = parse.statement(`
+        function Foo ({ children }) {
+          return children;
+        }
+      `);
+
+      expect(findFunctionReturn(def, predicate)).toBeDefined();
+    });
+
+    test('handles null return', () => {
+      const def = parse.statement(`
+        function Foo (props) {
+          return null;
+        }
```

Comment:
What worries me is that literaly any function that returns null will be detected as react component.
Same for the other usecase.